spec.html

```diff
@@ -41537,6 +41592,28 @@ <h1>
           <p>The parameter _iterable_ is expected to be an object that implements an @@iterator method that returns an iterator object that produces a two element array-like object whose first element is a value that will be used as a Map key and whose second element is the value to associate with that key.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-add-value-to-keyed-group" type="abstract operation">
+        <h1>
+          AddValueToKeyedGroup (
```

Comment:
It feels odd that `AddValueToKeyedGroup` AO is in the map section and the `GroupBy` AO is in the object section. They should probably both in the object section, since `AddValueToKeyedGroup` is used by `GroupBy`.

Reply:
Can do